backend/src/controllers/__test__/person.controller.test.ts

```diff
@@ -0,0 +1,36 @@
+import httpStatus from 'http-status';
+import databaseOperations from '../../test/db-handler';
+import { PersonModel } from '../../models/person.model';
+import app from '../../index';
+
+const supertest = require('supertest');
+
+beforeAll(async () => databaseOperations.connectDatabase());
+afterEach(async () => databaseOperations.clearDatabase());
+afterAll(async () => databaseOperations.closeDatabase());
+
+const requestPersonData:PersonModel = {
+  first_name: 'new person',
+  last_name: 'hellob',
+  interests: ['a', 'b'],
+  organisation: 'helloc',
+  time_added: new Date('2022-01-01'),
+  how_we_met: 'helloe',
+  birthday: new Date('2002-12-12'),
+  encounters: [] as any,
+};
+
+describe('person ', () => {
+  it('can be created correctly', async () => {
+    await supertest(app).post('/api/persons/create')
```

Comment:
I feel like this function is testing two things:
* If a person can be created properly
* If all people can be retrieved from the db

Would it be better to separate these tests out?

Reply:
answered in other comment (accidentally):

> Yes you are right. For this PR I was planning to just set up an example so anyone can refer to it when they make a test. I might leave it like that so that tests are written after this PR is merged. But definitely separate the test cases out like you said when you create the tests.